src/test/java/org/cirdles/topsoil/app/TopsoilMainWindowTest.java

```diff
@@ -31,5 +33,17 @@ public void start(Stage stage) throws Exception {
         stage.setScene(scene);
         stage.show();
     }
-    
+
+//    @Test
+//    public void testCreateTab() {
+//        int size = ((TabPane) lookup("#dataTableTabPane").queryFirst()).getTabs().size();
+//        clickOn("Create Data Table");
+//        verifyThat("#dataTableTabPane",
+//                (TabPane tabPane) -> tabPane.getTabs().size() > 0);
+//    }
```

Comment:
I'm fine with forgetting about trying to test this with as much time as we've sunk into it, but let's just get rid of it altogether (no comments).